fdbserver/VersionedBTree.actor.cpp

```diff
@@ -6234,7 +6234,7 @@ class VersionedBTree {
 		//
 		// There's another case. When encryption domain is enabled, we want to make sure the root node is encrypted
 		// using the default encryption domain. An indication that's not true is when the first record is not
-		// is enabled, and the root node is not encrypted using the default
+		// is using dbBegin as key.
```

Comment:
This says "is not is using" but I think you mean "is not using", there's just an extra "is".